src/components/notifications/Webhook.vue

```diff
@@ -12,6 +12,21 @@
     </div>
 
     <div class="mb-3">
+        <label for="webhook-http-method" class="form-label">{{ $t("HTTP Method") }}</label>
+        <select
+            id="webhook-http-method"
+            v-model="$parent.notification.httpMethod"
+            class="form-select"
+        >
+            <option value="post">POST</option>
+            <option value="get">GET</option>
+        </select>
+        <div class="form-text">
+            {{ httpMethodDescription }}
```

Comment:
much simpler to read this way

```suggestion
            {{ $parent.notification.httpMethod === 'get' ? $t("webhookGetMethodDesc") : $t("webhookPostMethodDesc") }}
```